Bdd/features/environment.py

```
@@ -10,6 +11,7 @@
 SYSLOG_NG_FULL_CONF = "Bdd/syslog-ng/syslog-ng-full.conf"
 SYSLOG_NG_CTL = "/var/lib/syslog-ng/syslog-ng.ctl"
 STORE_FILE_PATH = "/tmp/solidsyslog_store.dat"
+STORE_PATH_PREFIX = "/tmp/STORE"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/STORE"

(S108)